news/2018-10-10-beatmap-spotlights-summer-2018.md

```diff
@@ -0,0 +1,309 @@
+---
+layout: post
+title: "Beatmap Spotlights Summer 2018"
+date: 2018-10-10 06:00:00 +0000
+---
+
+Welcome back to a long-awaited rendition of this season's beatmap spotlights! We have lots of brilliant maps to showcase and some extra surprise for all of you!
+
+![](/wiki/shared/news/2018-10-10-beatmap-spotlights-summer-2018/spotlight-header.jpg)
+
+The beatmap charts for the Summer Spotlights can be found [here](https://osu.ppy.sh/p/chart?ch=SUMMER2018). Their respective packs are available at the [Beatmap Pack Listing](https://osu.ppy.sh/p/packlist?t=r) for easier access. **The Summer 2018 Spotlights rankings will end on 31th of December, 2018.**
+
+## Changes within the Spotlights!
+
+Without further ado let's get into some recent changes brought about to the spotlights program.
+
+You may have noticed that the release of spotlights have been somewhat inconsistent, but worry no longer! Spotlights will now be released 4 times per year, once per season, and organized as follows:
```

Comment:
full words are usually better than digits for small numbers like this. would just write `four`. comes up a few other times, but if seeing the scaling is important (ex. `receiving 3, 2, and 1 months of supporter` later in the post), you can keep as digits

Reply:
Changed the four but left the rest. Those are better that way

Reply:
"4 game modes" and "top 3"?